.pre-commit-hooks.yaml

```diff
@@ -17,8 +17,8 @@
     language: python
     types: [python]
 -   id: check-byte-order-marker
-    name: Check for byte-order marker
-    description: Forbid files which have a UTF-8 byte-order marker
+    name: 'check BOM - deprecated: use fix-byte-order-marker'
```

Comment:
I shortened the name here so it doesn't wrap -- I should probably add a test for this